main.css

```diff
@@ -0,0 +1,227 @@
+body {
+    font-family: Karla Regular;
+    font-weight: 400;
+    font-size: 14px;
+    margin: 0px;
+}
+
+header {

```

Comment:
Please don't style by element names (unless it's a global style, like the one for body). In bigger projects than yours it might cause issues when styling. Try to always style by classname.

Reply:
Have kept the below because it seems to have an auto margin I want to get rid of - or should I add a class name to the body? 

body {
     margin: 0px;
}